app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    problems_with_vaccines = 0
+    missing_masks = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            problems_with_vaccines += 1
```

Comment:
what about Outdated vaccine?
Also it's mentioned that:
`If at least one of the friend has problems with vaccines the function should return a message `"All friends should be vaccinated".`
Thus when you have such a friend, you can immediately return string